aten/src/ATen/native/native_functions.yaml

```diff
@@ -14720,6 +14720,13 @@
   dispatch:
     CompositeExplicitAutograd: wait_tensor
 
+- func: all_reduce_coalesced(Tensor[] self, str reduceOp, str tag, int[] ranks, int group_size) -> Tensor[]
+  # This should be changed to distributed but it requires changes all over the place to work
+  python_module: nn
```

Comment:
Since we're adding more and more of these, can we do this please?

Reply:
Ok, doing it on a separate PR to this one.

Reply:
PR for fixing this: https://github.com/pytorch/pytorch/pull/97793